warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexCachingIteratorJexl.java

```diff
@@ -1324,16 +1308,6 @@ protected boolean resumeFromPreviousCall() throws IOException {
             }
         }
 
```

Comment:
This check isn't necessary since we are assigning the necessary parts of the HDFSBackedSortedSet from the previous Ivarator to the current Ivarator -- i.e. ivaratorCacheDirs was used the first time around in setupRowBasedHdfsBackedSet